mmdt/templates/polls/all_results.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+    {% if is_admin %}
+        <h2>Admin View - All Questions - Voting Results</h2>
+    {% else %}
+        <h2>All Questions - Voting Results</h2>
+    {% endif %}
+    
+    <ul class="nav nav-tabs" id="questionTabs" role="tablist">
+        {% for question in all_questions %}
+            <li class="nav-item">
+                <a class="nav-link {% if forloop.first %}active{% endif %}" id="question-tab-{{ question.id }}" 
+                data-toggle="tab" href="#question-{{ question.id }}" role="tab" aria-controls="question-{{ question.id }}" 
+                aria-selected="{% if forloop.first %}true{% else %}false{% endif %}">{{ question.question_text }}</a>
+            </li>
+        {% endfor %}
+    </ul>
+
+    <div class="tab-content">
+        {% for question in all_questions %}
+            <div class="tab-pane fade {% if forloop.first %}show active{% endif %}" id="question-{{ question.id }}" role="tabpanel" aria-labelledby="question-tab-{{ question.id }}">
+                    <canvas id="resultChart_{{ question.id }}" width="500" height="500"></canvas>
+                    <script src="{% static 'assets/js/chart.min.js' %}"></script>
+                    <script src="{% static 'assets/js/chart.js' %}"></script>
+
+                    <script>
+                        document.addEventListener('DOMContentLoaded', function () {
+                            var ctx = document.getElementById('resultChart_{{ question.id }}').getContext('2d');
+                            var resultsData = {{ question.get_results_data|safe }};
+                    
+                            new Chart(ctx, {
+                                type: 'pie',
+                                data: {
+                                    labels: resultsData.labels,
+                                    datasets: [{
+                                        data: resultsData.data,
+                                        backgroundColor: [
+                                            '#f2c11d',
+                                            '#003399',
+                                            '#467df2'
```

Comment:
cause we set only 3 color, when we put the option in 4, the colour repeat and didn't know which one is which.


<img width="1073" alt="Screenshot 2024-03-01 at 8 23 03 PM" src="https://github.com/khinezarthwe/mmdt_web_app/assets/5201124/1162b710-0120-4a7d-bd88-8bc379e387bf">
<img width="1095" alt="Screenshot 2024-03-01 at 8 23 09 PM" src="https://github.com/khinezarthwe/mmdt_web_app/assets/5201124/91e72006-826e-429b-ae23-6ab3daa5b021">